aws/ansible/jenkins/jobs/cypress.groovy

```diff
@@ -27,6 +27,6 @@ job('Cypress-e2e-tests') {
         shell('CYPRESS_RECORD_KEY=${vault_cypress_record_key} '+
               'CYPRESS_NAP_LOGIN=${vault_cypress_nap_username} '+
               'CYPRESS_NAP_PASSWORD=${vault_cypress_nap_password} '+
-              '$(npm bin)/cypress run --browser chrome --record')
+              '$(npm bin)/cypress run --browser chrome --record false')
```

Comment:
Hm why does this pop up in the diff even though change was made also in 
https://github.com/finnishtransportagency/mmtis-national-access-point/pull/979/files ...